ansible/roles/layers-service/tasks/main.yml

```diff
@@ -0,0 +1,93 @@
+- include: ../../common/tasks/setfacts.yml
+
+- name: install Postgis and GDAL Execs
```

Comment:
Doesn't the postgresql role install postgis as well? Or this something different? 

It seems cleaner to have dedicated roles for software installs, so perhaps this could be moved to the postgresql role or a postgis role?